Sharks are really old - the earliest sharks lived about 450 million years ago.

Thats before trees evolved. Thats before there was enough oxygen in the atmosphere for combustion to occur.

Sharks are older than **fire**!

Thats a _galactic_ timescale - some stars exist for less time than sharks have lived. And – according to ChatGPT – 90% of visible stars would not have existed that long ago.

Lets see if we can validate that…

In [1]:
# !pip install astroquery
# !pip install pandas
from astroquery.gaia import Gaia
import pandas as pd

Some IP addresses of users launching heavy query showers have temporarily been disabled. Please contact the Gaia helpdesk (https://www.cosmos.esa.int/web/gaia/gaia-helpdesk) for advice. Workaround solutions for the Gaia Archive issues following the infrastructure upgrade: https://www.cosmos.esa.int/web/gaia/news#WorkaroundArchive


In [2]:
# Example: bright stars with astrophysical parameters (age, mass)
brightness = 6.5 # magnitude is a log scale; brighter = smaller.

query = f"""
SELECT TOP 20000
  g.source_id, g.ra, g.dec, g.phot_g_mean_mag,
  a.age_flame, a.mass_flame
FROM gaiadr3.gaia_source AS g
JOIN gaiadr3.astrophysical_parameters AS a ON g.source_id = a.source_id
WHERE g.phot_g_mean_mag <= {brightness}
  AND a.age_flame IS NOT NULL
"""
job = Gaia.launch_job(query)

In [3]:
table = job.get_results()
visible_stars_df = table.to_pandas()
len(visible_stars_df)

4193

In [5]:
age_threshold = 0.45 # 450 million years ago
shark_stars_df = visible_stars_df[visible_stars_df['age_flame'] > age_threshold]
len(shark_stars_df)

2598

In [6]:
len(shark_stars_df)/len(visible_stars_df)

0.6196041020748867

So - from a sample of 4193 visible stars which have been processed for FLAME age (roughly half of them), 2598 _probably_ would have existed 450 million years ago when sharks first evolved. 

That's about 62%.

Biased against very bright stars (which can saturate Gaia's detectors, so BP/RP may be bad or missing), and also distant, luminous (ie. old) giants.

I'm going to leave it there and say "ChatGPT got it wrong."

*Many* of the visible stars in the sky did not exist when the earliest sharks were swimming in the ocean – roughly a third.